addons/im_livechat/controllers/webclient.py

```diff
@@ -63,8 +63,7 @@ def _process_request_for_all(self, store: Store, name, params):
                 )
                 store.add_global_values(livechat_rule=Store.One(matching_rule))
             store.add_global_values(
-                livechat_available=matching_rule.action != "hide_button"
-                and bool(matching_rule._is_bot_configured() or channel.available_operator_ids),
+                livechat_available=matching_rule.action != "hide_button" and channel._is_livechat_available(),
```

Comment:
I don't think that's OK, let's say we have the following scenario:
- We have a channel were the /job page has a bot configured.
- We are on the contactus page, no agent is available and no chat bot is configured.

`is_livechat_available` checks if there is *a chat bot script for the given channel*, but it doesn't check if it's actually active on the page. As a result:
- Before this change the live chat button would not be shown.
- After this change, we will see the live chat button, even if we have no agent and no bot to handle the conversation.